pkgs/data/icons/papirus-icon-theme/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+    name = "papirus-icon-theme-${version}";
```

Comment:
We use 2 spaces to indent. See: https://github.com/NixOS/nixpkgs/blob/master/.editorconfig